bee-hive/cli/cli.py

```diff
@@ -12,13 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from cli.commands import Validate, Create, Run, Deploy
+from commands import Validate, Create, Run, Deploy
```

Comment:
latest pr merge adopted the approach of importing '.commands' or similar when referring to files within the current directory (since they might not need to be exported if only used locally).
For anything exported there's likely no effective difference but worth checking after rebase/merge
(applies to other files also)